src/utils/utils.ts

```diff
@@ -175,5 +175,12 @@ export function createInstanceofPredicate<T>(name: string, clazz: new (...args:a
 	} as any;
 }
 
+/**
+ * Returns whether the argument is an array, disregarding observability.
+ */
+export function isArrayLike(x: any): boolean {
+	return Array.isArray(x) || isObservableArray(x);
```

Comment:
The purpose of this method is to prevent people from trying to see an `ObservableArray` for the JS object it is. Alternative name: `quacksLikeAnArray` ;)